py/desispec/scripts/master_nea.py

```diff
@@ -0,0 +1,125 @@
+"""
+This script generates a master NEA (Noise Equivalent Area) file for a given camera.
+"""
+from   specter.psf.gausshermite  import  GaussHermitePSF
+from   desiutil.log import get_logger
+from   desispec.parallel import default_nproc as numprocesses
+from   functools import partial
+
+import numpy as np
+import argparse
+import sys
+import copy
+import astropy.io.fits as fits
+
+# https://github.com/desihub/desispec/issues/1006
+sampling           = 500
+
+wmin, wmax, wdelta = 3600., 9824., 0.8
+fullwave           = np.round(np.arange(wmin, wmax + wdelta, wdelta), 1)
+cslice             = {"b": slice(0, 2751, sampling), "r": slice(2700, 5026, sampling), "z": slice(4900, 7781, sampling)}
```

Comment:
Are the NEA files sensitive to the exact details of the wavelength grid?  This is likely to change again in the future, e.g. to extend the wavelength range further blue.  If it isn't sensitive to the sampling but perhaps is sensitive to the edges (e.g. due to being able to interpolate but not extrapolate) we should make sure this is a superset of what we might need in the future, e.g. 3520 - 9950.

Reply:
I'm requesting the wavelengths in the current spectro-perf. reductions.  I can update the nea files to the larger range, but I'm already seeing some breakdown at the current range and patching those with a median.  We argued the wavelength dependence was smooth so extrapolation may not be so bad.